.github/workflows/build-linux.yml

```diff
@@ -66,7 +66,7 @@ on:
 jobs:
   build-linux:
     name: build
-    runs-on: ubuntu-22.04
+    runs-on: ${{ inputs.platform == 'linux-aarch64' && 'ubuntu-22.04-arm' || 'ubuntu-22.04' }}
```

Comment:
Does this mean that the action will run on any runner that match this expression?
Is it possible that 'ubuntu-22.04' is still true when the platform is 'linux-aarch64'?

Reply:
It checks if the input parameter platform is set to 'linux-aarch64'. If so, it chooses the runner labeled 'ubuntu-22.04-arm', which is suitable for ARM64 architecture. Otherwise, it defaults to 'ubuntu-22.04', which is the standard x86_64 Ubuntu runner.
I can try `matrix` instead if this checking looks a bit confusing.

Reply:
Let's try matrix !

Reply:
Seems that it is more convenient to add a "runs-on" input to the build workflow. 
Fixed.